.dockerignore

```diff
@@ -1,2 +1 @@
-*
```

Comment:
why do we remove it?

Reply:
We need to copy / access all the go files as we now build inside the docker build step. (before it was build outside from docker and just the binary was copied)